lib/node_modules/@stdlib/stats/base/variancewd/benchmark/benchmark.js

```diff
@@ -39,12 +46,9 @@ var variancewd = require( './../lib/variancewd.js' );
 */
 function createBenchmark( len ) {
 	var x;
-	var i;
 
-	x = [];
-	for ( i = 0; i < len; i++ ) {
-		x.push( ( randu()*20.0 ) - 10.0 );
-	}
+	x = uniform( len, -10.0, 10.0, options ); // FIXED: Added `options`
+
```

Comment:
```suggestion
	var x = uniform( len, -10.0, 10.0, options );
```

Refactor to the above. That means deleting L48-49.